### PR TITLE
[Feature][Hitomi] added a Log In Page UI

### PR DESCRIPTION
**Pull request by HitomiP23**
added Log In Page UI

### CODE REVIEW

**Review by bit-Matt — APPROVED** [review]
LGTM

File: app/login.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+                    <img className="justify-center" src="gdsc_logo.png" alt="GDSC Logo" width={150} height={150}/>
+                </div>
+
+                <div className="text-white flex items-center justify-center mb-6 text-xs">GDSC-CPU</div>
+                <div className="w-full">
+                    <div className="mb-2 ">
+                        <input className="w-full rounded-sm p-2 text-xs" type="text" placeholder="Email/username"/>
+                    </div>
+                    <div className="mb-2 ">
+                    <input className="w-full rounded-sm p-2 text-xs" type="text" placeholder="Password"/>
```
Comment (cezro) [inline app/login.tsx:17]: Suggest putting `text-black`, input text not visible

File: app/login.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        <div className="w-full bg-[#07063D] flex items-center justify-center min-h-screen">
+            <div className="w-full lg:w-1/2 p-8 ">
+                <div className="justify-center flex items-center">
+                    <img className="justify-center" src="gdsc_logo.png" alt="GDSC Logo" width={150} height={150}/>
+                </div>
+
+                <div className="text-white flex items-center justify-center mb-6 text-xs">GDSC-CPU</div>
+                <div className="w-full">
+                    <div className="mb-2 ">
+                        <input className="w-full rounded-sm p-2 text-xs" type="text" placeholder="Email/username"/>
```
Comment (cezro) [inline app/login.tsx:14]: Suggest putting text-black, input text not visible

**Review by cezro — CHANGES_REQUESTED** [review]
other than few changes lgtm!

**Pull request merged**